samples/apps/SmartLauncher/Startup.cs

```diff
@@ -40,7 +40,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
             app.UseDefaultFiles();
-            app.UseStaticFiles(new StaticFileOptions { FileProvider = new EmbeddedFileProvider(Assembly.GetExecutingAssembly(), "SmartLauncher.wwwroot") });
+            app.UseStaticFiles(new StaticFileOptions { FileProvider = new EmbeddedFileProvider(Assembly.GetExecutingAssembly(), "Microsoft.Health.Internal.SmartLauncher.wwwroot") });
```

Comment:
This namespace was updated in code, it must be corrected here or the tests can't load the views